### PR TITLE
Onboard FirmwareAnalysis az cli commands

### PR DESCRIPTION
**Pull request by kalpeshbhatia000**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [x] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [x] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

File: src/firmwareanalysis/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Azure CLI Firmwareanalysis Extension #
+This is an extension to Azure CLI to manage Firmwareanalysis resources.
+
+## How to use ##
+Please add commands usage here.
```
Comment (necusjz) [inline src/firmwareanalysis/README.md:5]: please complete the basic usage scenarios to the users

File: src/firmwareanalysis/azext_firmwareanalysis/tests/latest/test_firmwareanalysis.py [hunk 2]
```diff
@@ -0,0 +1,140 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+import uuid
+import unittest
```
Comment (necusjz) [inline src/firmwareanalysis/azext_firmwareanalysis/tests/latest/test_firmwareanalysis.py:9]: seems useless

File: src/firmwareanalysis/README.md (hunk 1 above)
Comment (kalpeshbhatia000) [reply src/firmwareanalysis/README.md]: Added all commands in readme

File: src/firmwareanalysis/azext_firmwareanalysis/tests/latest/test_firmwareanalysis.py (hunk 2 above)
Comment (kalpeshbhatia000) [reply src/firmwareanalysis/azext_firmwareanalysis/tests/latest/test_firmwareanalysis.py]: remove these imports

**Review by necusjz — APPROVED** [review]

**Pull request merged**